ci/jenkins/README.md

```diff
@@ -1 +1,38 @@
 # Nephe CI: Jenkins
+
+This directory includes all the scripts required to run CI for nephe.
+
+We have static VM as jenkins node, when there is a pull request or daily build trigger the job. The code will be cloned on
+static VM, and it will call terraform to deploy dynamic VM and use ansible for system configuration such as installing docker,
+and then run tests on the dynamic VM, destroy the VM once the test succeeds or fails.
+
+## Terraform
+We create terrform tfvars file in nephe-ci.sh with jenkins credetials. `deploy.sh` is the script to create vm, and store the
+terraform state in a specified directory, which can be used to get VM information later. `destroy.sh` helps to detroy the VM
+and delete terraform workspace as well.
+
+For more details about terraform, please refer [here](https://www.terraform.io/docs)
+
+## Ansible
+Ansible will help us for system configuration after the VM installation finishes, all ansible tasks are in playbook/roles.
+For example, we add task to install docker in [playbook/roles/docker/tasks/main.yml](playbook/roles/docker/tasks/main.yml).
+
+Ansile also help to copy ssh keys to dynamic VM, by which we can run tests and collect log later.
+The ssh-keys are generated on static VM if nephe-ci checks there is no ssh-keys there.
+## jenkins job
+We use jenkins job-builder to manage jenkins jobs, the codes are in the directory jobs. There are four files:
+
+* **job-templates.yaml**: If you need several jobs defined that are nearly identical, except perhaps in their names,
```

Comment:
This may not be required.   Can you please take a look at https://github.com/antrea-io/antrea/blob/main/ci/jenkins/README.md as reference.

So it would look something like this.

# Introduction

# Jenkins Deployment

  ## Teraform
  
  ## Ansible
  
  ## Jenkins Job
  
  ### Requirements
  
  ### Jenkins job updater
 
# List of jenkins job

Reply:
done